snmp/datadog_checks/snmp/data/profiles/_base_cisco.yaml

```diff
@@ -32,6 +32,8 @@ metrics:
         name: cpmCPUMemoryUsed
       - OID: 1.3.6.1.4.1.9.9.109.1.1.1.1.13
         name: cpmCPUMemoryFree
+      - OID: 1.3.6.1.4.1.9.9.109.1.1.1.1.7
+        name: cpmCPUTotalEntry
```

Comment:
This isn't the correct name, this is `cpmCPUTotal1minRev`. The metadata is also incorrect in consequence.